### PR TITLE
Android: Add support for dynamic peripherals

### PR DESCRIPTION
**Pull request by AbandonedCart**

### CODE REVIEW

File: shell/android-studio/reicast/src/main/jni/src/Android.cpp [hunk 1]
```diff
@@ -265,7 +292,11 @@ void os_SetupInput()
 	for (int i = 0; i < 3; i++)
 	{
 		if (add_controllers[i])
-			mcfg_CreateController(i+1, MDT_None, MDT_None);
+			mcfg_CreateController(
+                    i + 1,
+                    GetMapleDeviceType(controller_periphs[i][0], i + 1),
```
Comment (AbandonedCart) [inline shell/android-studio/reicast/src/main/jni/src/Android.cpp]: Yes, I know these are one off

Comment (baka0815) [reply shell/android-studio/reicast/src/main/jni/src/Android.cpp]: Yeah, because of add_controllers starting wirh player 2. :)

**Comment by AbandonedCart** [comment]
*first steps to dynamic peripheral support*
Breathe. It's not being merged as is. Holy %&#!...

**Comment by baka0815** [comment]
I left some comments on the commit.
Sorry, for messing that up...

**Comment by AbandonedCart** [comment]
Without a connected UI, you are safe to assume it isn't done

**Pull request merged**